linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analysis.scala

```diff
@@ -216,8 +215,6 @@ object Analysis {
 
   def logError(error: Error, logger: Logger, level: Level): Unit = {
     val headMsg = error match {
-      case MissingJavaLangObjectClass(_) =>
-        "Fatal error: java.lang.Object is missing"
```

Comment:
@sjrd, if you feel we need to keep this error, we have two alternatives:

- Emit it in addition when the lookup of `ObjectClass` returns a `nonExistentClass`
- Special case `ClassInfo#link` to emit a different error if `className == ObjectClass` (there is precedent for `ClassInfo` to behave differently if it is jlO, so it's not that bad).

Reply:
The special error was only there because I thought it would crash the rest of the Analyzer if `Object` was missing. If it doesn't crash, it's fine.

Perhaps add a test where `jl.Object` is missing but there actually is another class? Currently the only test we have has *no* class at all.